### PR TITLE
[MOD-2404] : iOS overlay not visible on iPhone

### PR DESCRIPTION
**Pull request by vijaysingh-axway**
https://jira.appcelerator.org/browse/MOD-2404

### CODE REVIEW

**Comment by hansemannn** [comment]
Please use an „else“ statement to make the method available in the module as well, so devs can use it today already. We can remove it once the minimum SDK gets bumped to 7.2:0.

**Comment by vijaysingh-axway** [comment]
Good idea. Done!

File: ios/Classes/TiBarcodeModule.mm [hunk 1]
```diff
@@ -663,4 +652,282 @@ - (void)decoder:(Decoder *)decoder failedToDecodeImage:(UIImage *)image usingSub
...
+  
+  resultCenter.x += resultBounds.origin.x + viewBounds.origin.x;
+  resultCenter.y += resultBounds.origin.y + viewBounds.origin.y;
+  
+  [subView setAutoresizingMask:resultMask];
+  [subView setCenter:resultCenter];
+  [subView setBounds:resultBounds];
+}
+
+CGSize SizeConstraintViewWithSizeAddingResizing(LayoutConstraint *constraint, NSObject<LayoutAutosizing> *autoSizer, CGSize referenceSize, UIViewAutoresizing *resultResizing)
```
[the remark this answers is not in the document]
Comment (vijaysingh-axway) [reply ios/Classes/TiBarcodeModule.mm]: Yes. These methods are in SDK but we have issue in accessing them.

**Comment by hansemannn** [comment]
@vijaysingh-axway PR merged, great job! Is the release you drafted the one that contains the latest fixes? I just released it.

**Comment by vijaysingh-axway** [comment]
Correct. Thanks1

**Pull request merged**